### ISSUE TITLE
NFTs aren't loading in MyMeta

### ISSUE THREAD

**Issue by dysbulic** [MetaFam/TheGame#1102]
### What happened?

The requests to OpenSea to load a user's NFTs are being rejected with a HTTP 403 forbidden.

I don't know if it is an issue of the access token being canceled, or it could be something deeper.

One potential issue is the environment variable used to provide the token is [`NEXT_OPENSEA_API_KEY`](//github.com/MetaFam/TheGame/blob/f5295c324244f2973ea419c588e784c142a8091b/packages/web/config.ts#L16). According to [the documentation](//nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser), variables being accessed by the browser need to be prefaced with `NEXT_PUBLIC_`.

### What did you expect to happen?

On a user's MyMeta profile page, the "NFT Gallery" should have content in it.

### How can we reproduce the problem (as minimally as possible)?

Visit [test.metagame.wtf/me](//test.metagame.wtf/me) and view the NFT Gallery.

### Is there anything else we need to know?

You can see the error for the rejected connection to OpenSea in the browser console.

**Comment by dan13ram** [comment]
I think we should move away from opensea and use subgraphs to fetch nfts on various networks.

**Comment by dan13ram** [comment]
@firattale Are you working on this or Can I take a look ?

**Comment by firattale** [comment]
hi @dan13ram , I am checking it today, but any help would be appreciated

**Comment by dysbulic** [comment]
> > [V]ariables being accessed by the browser need to be prefaced with `NEXT_PUBLIC_`.
> 
> This could be it, though it seems unlikely that this has changed.

Yeah, I checked the blame & the OpenSea token was added over a year ago & has not changed.

> 1. Seems like we should not be exposing our API key in the browser

The browser is making the request, so there's not much choice. We could proxy the request like I'm doing to protect the [web3.storage](//web3.storage) API key, but it adds overhead & complexity.

> 2. The API Key is currently not in the header of the GET request

`opensea-js` went from `1.2.1` to `1.2.7` last month. That seems to correlate to when it broke. @firattale, you might try reverting & see if that resolves the 403.
